src/components/Shared/MessagesList.tsx

```diff
@@ -85,27 +86,36 @@ const ConversationBeginningNotice: FC = () => (
 
 interface MessageListProps {
   messages: Message[];
+  fetchNextMessages: () => void;
 }
 
-const MessagesList: FC<MessageListProps> = ({ messages }) => {
+const MessagesList: FC<MessageListProps> = ({ messages, fetchNextMessages }) => {
```

Comment:
Is it possible to reuse the `useInView` hook that powers pagination in places like the Explore feed instead of introducing a new `InfiniteScroll` dep?

https://github.com/lensterxyz/lenster/blob/c76564fb8672bf3a99f8bd9295e76cb8354dcad8/src/components/Explore/Feed.tsx#L42-L54

Reply:
Yea we don't want to add external dependency for the same purpose 🙏🏼

Reply:
This seems to not work well, I tried. Will further look into it

Reply:
Resolved this

Reply:
Amazing @bhavya2611

Reply:
@bhavya2611 We need to move back to `react-infinite-scroller` we came to know `usInview` is more buggy and complex to maintain

We suggest you revert back the changes to use `react-infinite-scroller`

Sorry for the inconvenience happened 🙏🏼 

Make sure to rebase the branch 😊